### ISSUE TITLE
Add text link to Works by artist you follow

### ISSUE THREAD

**Issue by katarinabatina** [artsy/eigen#162]
![image](https://cloud.githubusercontent.com/assets/6482926/6175109/f72dd21c-b2c1-11e4-95b6-fb79822ce2b5.png)

leads to martsy view https://www.artsy.net/works-for-you

**Comment by orta** [comment]
FeaturedLink Set, for admins only, http://admin.artsy.net/set/5308e7be9c18db75fd000343

**Comment by orta** [comment]
Hrm, this is harder than expected actually. You only want to show that for logged in users.

**Comment by katarinabatina** [comment]
yup

**Comment by katarinabatina** [comment]
if this is proving difficult we could surface the sign up ui for trial users

**Comment by orta** [comment]
could add a new featuredSet just for logged in users?

**Comment by katarinabatina** [comment]
sure, whatever makes this easiest to implement

**Comment by orta** [comment]
There's a the ability to add this now, for the next build :+1:

**Comment by 1aurabrown** [comment]
On iPhone, 'featured works' is a martsy view and on iPad it's native. will this be native on both devices?

**Comment by 1aurabrown** [comment]
I added the url to the featured links set. on iPad, do we want to do the same thing that we do for featured works, which is to show them in a native view contained in the feed?

There's a separate issue we should consider, which is that if a user is initially using the app as a trial user and later logs in, we should then update the feed view to add this link (or the native view containing the works for you). Inversely, I've noticed a problem when using the dev menu log out button, where if I am logged in and viewing user-specific content and then log out, somehow requests to the API that require a current user still succeed (though `[User currentUser]` is appropriately nil). I'm unclear on what reference to the user is not being removed properly when hitting log out.
